ocs_ci/ocs/resources/storage_cluster.py

```diff
@@ -498,3 +499,118 @@ def get_all_storageclass():
         )
     ]
     return storageclass
+
+
+def check_rebalance_occur_after_expand(old_osds):
+    """
+    The function check data balance is initiating after add capacity to cluster. The function need 2 out 3 rounds
+    the old osd that had data will be decreased and new osd data will be increased.
```

Comment:
plz add empty line between description and args

Reply:
done